backend/inspirehep/files/api/s3.py

```diff
@@ -22,6 +23,13 @@ def __init__(self, client, resource, config=None):
             config = TransferConfig(max_concurrency=1, use_threads=False)
         self.config = config
 
+    @property
+    def public_file_path(self):
+        return urljoin(
+            f"{current_app.config['PREFERRED_URL_SCHEME']}://{current_app.config['SERVER_NAME']}",
```

Comment:
can't we use https://github.com/inspirehep/inspirehep/blob/e33f5c2f4d55573ff115f154169664020f7883e8/backend/inspirehep/utils.py#L17 ?